runtime/bindings/python/tests/test_inference_engine/test_infer_request.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import os
+import pytest
+
+from openvino import Core, Blob, TensorDesc, StatusCode
+
+
+def image_path():
+    path_to_repo = os.environ["DATA_PATH"]
+    path_to_img = os.path.join(path_to_repo, "validation_set", "224x224", "dog.bmp")
```

Comment:
why do we need image? Tests are work on dummy data

Reply:
discussed offline. Created ticket: 67090